### PR TITLE
loadObjects to use all associations from previously run query

### PR DESCRIPTION
**Pull request by akuppusamy-wci**

### CODE REVIEW

File: src/main/java/gov/nih/nci/evs/api/service/AbstractStardogLoadServiceImpl.java [hunk 1]
```diff
@@ -464,6 +476,22 @@ public void loadObjects(
     logger.info("Done loading Elastic Objects!");
   }
```
Comment (gaaliciA1990) [inline src/main/java/gov/nih/nci/evs/api/service/AbstractStardogLoadServiceImpl.java:478]: Let's add documentation on what this class does (converts what to an AssociationEntry)

File: src/main/java/gov/nih/nci/evs/api/util/HierarchyUtils.java [hunk 2]
```diff
@@ -704,4 +704,8 @@ public Map<String, List<Association>> getInverseAssociationMap() {
   public void setInverseAssociationMap(Map<String, List<Association>> inverseAssociationMap) {
     this.inverseAssociationMap = inverseAssociationMap;
   }
+
```
Comment (gaaliciA1990) [inline src/main/java/gov/nih/nci/evs/api/util/HierarchyUtils.java:707]: Add documentation for the getter here too.

**Review by gaaliciA1990 — COMMENTED** [review]
Missing some documentation for new methods, but the implementation makes sense for what we're trying to achieve with adding the associations

**Comment by gaaliciA1990** [comment]
@akuppusamy-wci  26 tests are failing on my machine, have you verified all tests are passing on yours?

**Review by peter-va — APPROVED** [review]

**Review by bcarlsenca — APPROVED** [review]
OK merging in so we can devreset and re-test

**Pull request merged**